internal/plugin/server.go

```diff
@@ -57,10 +57,11 @@ type nvidiaDevicePlugin struct {
 	cdiHandler          cdi.Interface
 	cdiAnnotationPrefix string
 
-	socket string
-	server *grpc.Server
-	health chan *rm.Device
-	stop   chan interface{}
+	socket    string
+	server    *grpc.Server
+	healthy   chan *rm.Device
```

Comment:
Instead of introducing a new channel, is there any benefit in having a single channel that accepts a device and the desired status?